streams/integration-tests/src/test/java/org/apache/kafka/streams/integration/utils/EmbeddedKafkaCluster.java

```diff
@@ -140,6 +140,13 @@ public EmbeddedKafkaCluster(final int numBrokers,
         this.time = new MockTime(mockTimeMillisStart, mockTimeNanoStart);
     }
 
+    public static EmbeddedKafkaCluster withStreamsRebalanceProtocol(final int numBrokers) {
```

Comment:
nit: maybe either add a `Properties` parameter or an overload with `Properties`, we'll need that if we want to test using standby replicas

Reply:
Done